Wine_project_Tanya_new.ipynb

```diff
@@ -0,0 +1,1979 @@
+{
```

Comment:
<p>Put this in a markdown cell rather than printing it in python (press m when you create a new cell and are in command mode after pressing esc)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-TataAndBigData/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>